### PR TITLE
document Operator crash + workaround

### PR DESCRIPTION
**Pull request by taroface**
Fixes https://github.com/cockroachdb/cockroach-operator/issues/647

### CODE REVIEW

**Review by keith-mcclellan — CHANGES_REQUESTED** [review]
LGTM but we need this in the openshift docs too please, replacing kubectl with oc

**Comment by taroface** [comment]
@keith-mcclellan Added a similar callout to the OpenShift doc. Can you please have another look?

**Review by keith-mcclellan — APPROVED** [review]
LGTM

File: v21.1/deploy-cockroachdb-with-kubernetes-openshift.md [hunk 1]
```diff
@@ -79,10 +79,10 @@ This article assumes you have already installed the OpenShift Container Platform
     $ oc get pods
     ~~~
 
-	~~~
-	NAME                                  READY   STATUS    RESTARTS   AGE
-	cockroach-operator-65c4f6df45-h5r5n   1/1     Running   0          51s
-	~~~
+		~~~
```
Comment (gemma-shay) [inline v21.1/deploy-cockroachdb-with-kubernetes-openshift.md:82]: formatting is broken in the preview here, but lgtm otherwise!

Comment (taroface) [reply v21.1/deploy-cockroachdb-with-kubernetes-openshift.md]: Thanks for catching this! No idea why the formatting doesn't work this way, but I was able to fix it.

**Pull request merged**